libs/docs/core/segmented-button/examples/segmented-button-complex-example/segmented-button-complex-example.component.ts

```diff
@@ -1,14 +1,15 @@
 import { NgFor } from '@angular/common';
 import { Component } from '@angular/core';
 import { FormsModule } from '@angular/forms';
+import { FocusableListModule } from '@fundamental-ngx/cdk';
```

Comment:
do not import anything from the `cdk` directly, use `cdk/utils` secondary entrypoint and do you need `Module`?